velox/functions/prestosql/types/tests/BingTileTypeTest.cpp

```diff
@@ -40,4 +40,36 @@ TEST_F(BingTileTypeTest, serde) {
   testTypeSerde(BINGTILE());
 }
 
+TEST_F(BingTileTypeTest, packing) {
+  uint64_t tile_x123_y111_z7 = bingTileCoordsToInt(123, 111, 7);
```

Comment:
nit: can you please update the variable names to use camel case to conform to velox's [naming convention](https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions)